po/r-novice-gapminder.ja.po

```diff
@@ -544,30 +544,30 @@ msgid ""
 "  licensor endorses you or your use."
 msgstr ""
 "  your work is derived from work that is Copyright © Software\n"
-"  Carpentry and, where practical, linking to\n"
-"  http://software-carpentry.org/), provide a [link to the\n"
-"  license][cc-by-human], and indicate if changes were made. You may do\n"
-"  so in any reasonable manner, but not in any way that suggests the\n"
-"  licensor endorses you or your use."
+"  Carpentry and, where practical, linking to http://software-carpentry.org/), \n"
+"  [ライセンスへのリンク](https://creativecommons.org/licenses/by/4.0/deed.ja)を\n"
+"  提供し、変更があったらその旨を示さなければなりません。\n"
+"  これらは合理的であればどのような方法で行っても構いませんが、\n"
+"  許諾者があなたやあなたの利用行為を支持していると示唆するような方法は除きます。"
```

Comment:
```suggestion
"  著作権©ソフトウェアカーペントリーである作品から派生していることに言及して、\n"
"  そして実用的な場合はhttp://software-carpentry.orgにリンクしていることを述べます), \n"
"  [ライセンスへのリンク](https://creativecommons.org/licenses/by/4.0/deed.ja)を\n"
"  提供し、変更があったらその旨を示さなければなりません。\n"
"  これらは合理的であればどのような方法で行っても構いませんが、\n"
"  許諾者があなたやあなたの利用行為を支持していると示唆するような方法は除きます。"
```

Reply:
I've gone ahead and changes this in `license` branch locally since multi-line suggestions are in beta and don't seem to support this.